superset-frontend/eslint/eslint-plugin-literal-colors/index.js

```diff
@@ -0,0 +1,132 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * @fileoverview Rule to warn about literal colors
+ * @author Apache
+ */
+
+const cssData = require('mdn-data').css;
+const _ = require('lodash');
+
+const ALLOWED_VALUES = [
+  'auto',
+  'inherit',
+  'initial',
+  'unset',
+  'none',
+  'revert',
+  'invert',
+];
+
+function getObjectExpressionPropertyNodes(
+  objectExpressionNode,
+  propertyNodes = [],
+) {
+  _.each(objectExpressionNode.properties, propertyNode => {
+    propertyNodes.push(propertyNode);
+
+    if (
+      propertyNode.value &&
+      propertyNode.value.type &&
+      propertyNode.value.type === 'ObjectExpression'
```

Comment:
use optional chaining to simplify expression
```suggestion
      propertyNode?.value?.type === 'ObjectExpression'
```